optimum/onnxruntime/modeling_seq2seq.py

```diff
@@ -686,7 +686,7 @@ class ORTModelForConditionalGeneration(ORTModel, ABC):
         config ([`PretrainedConfig`]):
             Instance of the configuration associated to the model. Initializing with a config file does
             not load the weights associated with the model, only the configuration.
-        use_io_binding (`bool`):
+        use_io_binding (`bool`, *optional*, defaults to `None`):
```

Comment:
```suggestion
        use_io_binding (`Optional[bool]`, defaults to `None`):
```

aligning on https://huggingface.slack.com/archives/C02P0559X9S/p1669628754896019